lib/plugins/stats.js

```diff
@@ -5,17 +5,19 @@ module.exports = async robot => {
   // Cache of stats that get reported
   const stats = {installations: 0, popular: []}
 
-  // Setup /stats endpoint to return cached stats
-  robot.router.get('/stats', (req, res) => {
-    res.json(stats)
-  })
-
   // Refresh the stats when the plugin is loaded
-  await refresh()
+  const initializing = refresh()
 
   // Refresh the stats on an interval
   setInterval(refresh, REFRESH_INTERVAL)
 
+  // Setup /stats endpoint to return cached stats
+  robot.router.get('/stats', async (req, res) => {
```

Comment:
I’d suggest to prefix all probot core routes with `/probot`, so I’d do `/probot/stats` for this.

The reason is that I expect/hope that we will have a  way to add custom UI soon and then folks can use all routes for their own web apps, only the ones prefixed with `/probot/` would be reserved.